### DISCUSSION TITLE
Helper utility

### DISCUSSION THREAD

**Discussion by guest73** [BurntSushi/ripgrep#1643] [General] [answered]
Is there anything like [Sack](https://github.com/sampson-chen/sack) for ripgrep? Cannot find any on the internet.

**Comment by BurntSushi** [comment, marked as answer]
I don't know of any. If Sack supports `ag`, then you can probably just replace `ag` with `rg` and it might mostly work. You might need to tweak some flag names, but that should be it.

**Reply by guest73** [reply]
Well, I am not sure if it's that easy, there is an issue to add a [support](https://github.com/sampson-chen/sack/issues/41) for ripgrep. Maybe I'll give it a try when I have time. Thanks for response.
